tests/scripts/local_monitoring_tests/rest_apis/test_create_metric_profile.py

```diff
@@ -25,6 +25,12 @@
 from helpers.utils import *
 from helpers.list_metric_profiles_validate import *
 from helpers.list_metric_profiles_without_parameters_schema import *
+from pathlib import Path
+
+current_directory = Path(__file__).resolve().parent
```

Comment:
@shreyabiradar07 Please add a common function in utils to get the metric profile dir

Reply:
Updated common function - `get_metric_profile_dir()` in `utils.py`